packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.html

```diff
@@ -18,8 +21,14 @@
 
 <amplify-slot name="sign-in-footer" [context]="context">
   <div data-amplify-footer>
-    <button amplify-button fontWeight="normal" size="small" variation="link" fullWidth="true"
-      (click)="authenticator.toResetPassword()">
+    <button
+      amplify-button
+      fontWeight="normal"
```

Comment:
Maybe I miss something here. Shouldn't we avoid using styling props as well? This will block customers using strict CSP rules as well.

Reply:
Yes you are correct, and this will still throw errors for the strictest of CSP rules, but it does work with angular's recommended csp rule https://angular.io/guide/security#content-security-policy